automatic_push/ietf_push.py

```diff
@@ -182,25 +173,62 @@ def _update_rfc_files_locally(self, new_files: list[str], diff_files: list[str])
         if self.send_message:
             self.logger.info('new or modified RFC files found. Sending an E-mail')
             mf = message_factory.MessageFactory()
-            if self.repo.repo.index.diff(None, paths=self.rfc_dir):
+            for _ in self.repo.repo.index.diff(None, paths=self.rfc_dir):
```

Comment:
why is this better than an if?

Reply:
```repo.index.diff()``` returns a generator, so it's enough to check that at least one file is in diff